run.py

```diff
@@ -535,7 +550,7 @@ def run(args):
                     ceph_pkgs = requests.get(
                         base_url + "/Tools/Packages/", verify=False
                     )
-                elif cloud_type == "openstack":
+                elif cloud_type == "openstack" or cloud_type == "baremetal":
```

Comment:
The idea of using `cloud_type` may help us help. However, am not sure if it is the right way to differentiate a host environment.... Let's table this for discussion.